marshal/oracle/segment_advisors.go

```diff
@@ -43,9 +46,5 @@ func SegmentAdvisor(cmdOutput []byte) []model.OracleDatabaseSegmentAdvisor {
 		}
 	}
 
-	if len(segmentadvisors) == 0 {
-		return segmentadvisors
-	}
-
-	return segmentadvisors[2:]
+	return segmentadvisors
```

Comment:
Have you checked that don't broke anything?

Reply:
Yes, I tested it.
https://github.com/ercole-io/ercole-agent/pull/194/files#diff-76b0586deb5f31de8c4706ef95019bb3